### PR TITLE
Bump versions in the synthesis workflow

### PR DESCRIPTION
**Pull request by wizmer**
- placement-algorithm
- region-grower
- TNS

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-tmd/package.py [hunk 1]
```diff
@@ -10,10 +10,9 @@ class PyTmd(PythonPackage):
     """Python library for the topological analysis of neurons"""
 
     homepage = "https://github.com/BlueBrain/TMD"
-    url = "https://pypi.io/packages/source/t/tmd/tmd-2.0.3.tar.gz"
+    url = "https://pypi.io/packages/source/t/tmd/tmd-2.0.8.tar.gz"
 
-    version('2.0.4', sha256='b1709f36964d11dd555a35f99dfa26187ce2ef98a56f07eaa4805da944e86652', preferred=True)
-    version('2.0.3', sha256='ffde39a2fa6221a093ff66390a205149348e97a459aba1ab8f04998070e00dc1')
+    version('2.0.8', sha256='b1709f36964d11dd555a35f99dfa26187ce2ef98a56f07eaa4805da944e86652')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-tmd/package.py]: Everything else is tagged :)

Now why is the SHA the same, but the version bumped?

**Review by matz-e — DISMISSED** [review]
Looks good, still think you could leave old versions in the files (I'm OK with ditching `@space2`, though)

File: var/spack/repos/builtin/packages/py-tmd/package.py (hunk 1 above)
Comment (wizmer) [reply var/spack/repos/builtin/packages/py-tmd/package.py]: This one comes from PyPi, hence the tags.
I have updated the new SHA.
And I have changed my mind about keeping `@space2` in fact :)

Comment (matz-e) [reply var/spack/repos/builtin/packages/py-tmd/package.py]: Ah, sorry, didn't realize that. Thanks for fixing the sha!

**Review by matz-e — DISMISSED** [review]

**Comment by matz-e** [comment]
Retest this please

**Comment by pramodk** [comment]
@wizmer : can you take a look at failure : https://bbpcode.epfl.ch/ci/blue/organizations/jenkins/hpc.spack-deployment.pull-requests/detail/hpc.spack-deployment.pull-requests/459/tests ?

**Comment by wizmer** [comment]
Retest this please

**Review by matz-e — APPROVED** [review]

**Pull request merged**